### DISCUSSION TITLE
Run actions/upload-artifact@v3 Error: No files were found with the provided path: output.zip. No artifacts will be uploaded.

### DISCUSSION THREAD

**Discussion by mattmaximum** [philosowaffle/peloton-to-garmin#542] [General]
For github actions, it's throwing this error and is blowing up my email inbox. My workouts are appearing fine though.

Run actions/upload-artifact@v3
`Run actions/upload-artifact@v3
/usr/bin/docker exec  4925efee46abfd6c197[8](https://github.com/mattmaximum/peloton-to-garmin/actions/runs/6386032191/job/17331993421#step:9:9)a3535f475f0384eb08c70da6b056cfb3fa7d7c[9](https://github.com/mattmaximum/peloton-to-garmin/actions/runs/6386032191/job/17331993421#step:9:10)990ef sh -c "cat /etc/*release | grep ^ID"
Error: No files were found with the provided path: output.zip. No artifacts will be uploaded.`

**Comment by aclarke-GH** [comment]
running into the same issue

**Comment by wesraynbirdtilbury** [comment]
`zip error: Nothing to do! (try: zip -r -j output.zip . -i /app/output)
Error: Process completed with exit code 12.`

I'm getting this - @philosowaffle any suggestions?

**Comment by philosowaffle** [comment]
Just pushed an updated version of the [workflow](https://github.com/philosowaffle/peloton-to-garmin/blob/master/.github/workflows/sync_peloton_to_garmin.yml). Give it a go and see if that resolves the error. (after syncing don't forget to update [this line](https://github.com/philosowaffle/peloton-to-garmin/blob/master/.github/workflows/sync_peloton_to_garmin.yml#L23) back to console-latest)

**Reply by wesraynbirdtilbury** [reply]
Thanks @philosowaffle - what do you mean by '(after syncing don't forget to update [this line](https://github.com/philosowaffle/peloton-to-garmin/blob/master/.github/workflows/sync_peloton_to_garmin.yml#L23) back to console-latest)'

I'm seeing this, is this correct?
```image:  philosowaffle/peloton-to-garmin:stable```

**Reply by philosowaffle** [reply]
No it needs to be `:console-latest`. More info [here](https://github.com/philosowaffle/peloton-to-garmin/issues/526#issuecomment-1742214948).

**Comment by mattmaximum** [comment]
This fixed it for me, thank you.
